lib/Dialect/Comb/CombFolds.cpp

```diff
@@ -891,6 +891,104 @@ OpFoldResult OrOp::fold(ArrayRef<Attribute> constants) {
       constants, [](APInt &a, const APInt &b) { a |= b; });
 }
 
+/// Merge concats in an or op when they have non-conflicting fields.
```

Comment:
Plz add an example in the comment, it will make it easier to know at-a-glance what sort of thing this is doing.

Reply:
Yep, will do.

Reply:
Added a pair of examples to this in my latest push.